.github/workflows/ci.yml

```diff
@@ -16,5 +16,19 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@8c0edbc76e98fa90f69d9a2c020dcb50019dc325
+        with:
+          install: true
+
+      - name: Build and push
+        uses: docker/build-push-action@c56af957549030174b10d6867f20e78cfd7debc5
+        with:
+          context: .
+          push: false
```

Comment:
This prevents the image from being pushed to docker hub.
As a consequence, it'll only be built.